modules/linux

```diff
@@ -172,6 +172,15 @@ $(build)/$(BOARD)/bzImage: $(build)/$(linux_dir)/.build
 	@touch $@ # force a timestamp update
 	@sha256sum "$@" | tee -a "$(HASHES)"
 
+# Build kernel second time, not that initrd is built.
```

Comment:
not -> now

Reply:
Fixed and force pushed (in case you missed it).